include/rtweekend.hpp

```diff
@@ -55,12 +54,12 @@ uint32_t toseed(vec const& val1, vec const& val2) {
   std::memcpy(&y2, &val2.y(), sizeof(uint32_t));
   std::memcpy(&z2, &val2.z(), sizeof(uint32_t));
   uint32_t shifted1 = x1 << 26;
-  uint32_t shifted2 = (x2 & 63) << 21;
+  uint32_t shifted2 = (x2 & 31) << 21;
   uint32_t shifted3 = (y1 & 63) << 15;
-  uint32_t shifted4 = (y2 & 63) << 10;
-  uint32_t shifted5 = (z1 & 63) << 5;
-  uint32_t shifted6 = (z2 & 63);
-  return shifted1 ^ shifted2 ^ shifted3 ^ shifted4 ^ shifted5 ^ shifted6;
```

Comment:
Ah I read too quickly and was thinking to 63 for 5 bits...
Actually that seems like a good idea then to have an xor. Would more than 6 bits better than? 127 or 255?